types/wellknown/wellknown-tests.ts

```diff
@@ -1,6 +1,6 @@
 import * as wellknown from 'wellknown';
 wellknown.parse("POINT(1 2)");
```

Comment:
This one should be probably updated to:
```ts
wellknown.parse("POINT(1 2)"); // $ExpectType wellknown.GeoJSONGeometry | null
```

Reply:
For some reason this fails with
```
ERROR: 4:1  expect  TypeScript@next expected type to be:
  GeoJSONGeometry | null
got:
  GeoJSONPoint | GeoJSONMultiPoint | GeoJSONLineString | GeoJSONMultiLineString | GeoJSONPolygon | GeoJSONMultiPolygon | null
```

Reply:
Turns out it seems like $ExpectType her only recognizes a single type..

Reply:
Sorry for the fuss here, I was wrong (looked probably into wrong version). You're correctly on this `GeoJSONGeometry` is OK